### PR TITLE
XIVY-12550 Add Cms-Action to category path chooser

### PR DESCRIPTION
**Pull request by ivy-edp**
I have now integrated the new core action into the frontend. For this action, I used the CMS browser icon.

grafik

### CODE REVIEW

**Comment by ivy-lli** [comment]
I would move this open icon to the fieldset as control

File: packages/editor/src/components/parts/common/info/Information.tsx [hunk 1]
```diff
@@ -46,6 +47,15 @@ const Information = <T extends InformationConfig>({ config, update }: Informatio
     })
   ];
 
+  const getCMSUrl = (category: string) => {
+    let middlePart = category;
+    if (category.includes('<%=')) {
+      middlePart = 'MACRO';
+    }
+    console.log('/Categorie/' + middlePart + '/name');
```
Comment (ivy-lli) [inline packages/editor/src/components/parts/common/info/Information.tsx:55]: remove

File: packages/editor/src/components/parts/common/info/Information.tsx [hunk 2]
```diff
@@ -46,6 +47,15 @@ const Information = <T extends InformationConfig>({ config, update }: Informatio
     })
   ];
 
+  const getCMSUrl = (category: string) => {
+    let middlePart = category;
+    if (category.includes('<%=')) {
```
Comment (ivy-lli) [inline packages/editor/src/components/parts/common/info/Information.tsx:52]: was this supported in the old swt inscription? open a macro cms path?

**Review by ivy-lli — COMMENTED** [review]
As already commented, if you move this open action button to the fieldset you can remove a lot of the code here...

File: packages/editor/src/components/parts/common/info/Information.tsx (hunk 2 above)
Comment (ivy-edp) [reply packages/editor/src/components/parts/common/info/Information.tsx:52]: yes, in the old SWT Inscription, clicking the button while a macro was selected would add it to the CMS under Categories/MACRO/name.

Comment (ivy-lli) [reply packages/editor/src/components/parts/common/info/Information.tsx:52]: Sounds like a bug to me...
Or did bruno confirm this was a feature and was used?

Comment (ivy-edp) [reply packages/editor/src/components/parts/common/info/Information.tsx:52]: Yes, thats possible. I haven't asked him yet, as he was still on vacation last week, but I can definitely check with him.

**Comment by ivy-edp** [comment]
I can move the button to the Category-Fieldset but i just thought that it looks a little strange. What do you think?
grafik

**Comment by ivy-lli** [comment]
I prefer this solution a lot more. Because all such action live on the fieldset label, or collapsible level.

Change value of input actions -> beside input
Do something with value -> beside label

**Review by ivy-lli — APPROVED** [review]

**Pull request merged**